.github/workflows/conformance-ingress.yaml

```diff
@@ -0,0 +1,153 @@
+name: ConformanceIngress
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  minikube_version: 1.25.2
+  cilium_cli_version: v0.11.5
+  timeout: 5m
+  echo_server_image: k8s.gcr.io/ingressconformance/echoserver:v0.0.1@sha256:9b34b17f391f87fb2155f01da2f2f90b7a4a5c1110ed84cb5379faa4f570dc52
+
+jobs:
+  ingress-conformance-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+
+      - name: Install Cilium CLI
+        run: |
+          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
+          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+          rm cilium-linux-amd64.tar.gz{,.sha256sum}
+          cilium version
+
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          persist-credentials: false
+
+      - name: Set image tag
+        id: vars
+        run: |
+          if [ ${{ github.event.pull_request }} ]; then
+            SHA=${{ github.event.pull_request.head.sha }}
+          else
+            SHA=${{ github.sha }}
+          fi
+          echo ::set-output name=sha::${SHA}
+
+      - name: Create minikube cluster with multiple nodes
+        run: |
+          # Use at least 3 nodes, so that at least 2 worker nodes are available for scheduling
+          minikube start --network-plugin=cni --cni=false \
+            --docker-opt="default-ulimit=nofile=102400:102400" \
+            --nodes 3 \
```

Comment:
Is there any specific reason for using Minikube instead of Kind? I'm mostly wondering, since we already have a Kind conformance test, if it would make sense to reify the two so that we don't maintain multiple versions.

Reply:
> Minikube cluster (with multiple nodes enabled) is used due to the
native support for LB services (with minikube tunnel) without the
complicated steps to install metalLB like kind.

Yup, I tried with kind + metalLB but it's kind of tricky with steps and also unstable

Reply:
Ah sorry I missed that. Thanks!